CondCore/ESSources/plugins/CondDBESSource.cc

```diff
@@ -147,69 +148,73 @@ CondDBESSource::CondDBESSource(const edm::ParameterSet& iConfig)
 
   /*parameter set parsing
    */
-  std::string globaltag("");
-  if (iConfig.exists("globaltag")) {
-    globaltag = iConfig.getParameter<std::string>("globaltag");
-    // the global tag _requires_ a connection string
-    m_connectionString = iConfig.getParameter<std::string>("connect");
-
-    if (!globaltag.empty()) {
-      edm::Service<edm::SiteLocalConfig> siteLocalConfig;
-      if (siteLocalConfig.isAvailable()) {
-        if (siteLocalConfig->useLocalConnectString()) {
-          std::string const& localConnectPrefix = siteLocalConfig->localConnectPrefix();
-          std::string const& localConnectSuffix = siteLocalConfig->localConnectSuffix();
-          m_connectionString = localConnectPrefix + globaltag + localConnectSuffix;
-        }
+  std::string globaltag = iConfig.getParameter<std::string>("globaltag");
```

Comment:
These parameters could be probably now better provided in the constructor initializer list (where they are currently initialized to some default hardcoded value), given that they are now automatically defined by the fillDescriptions method

Reply:
Done in the last push.